package/netevent/package

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(netevent)
+pkgdesc="Input-Event device cloning utility"
+url="https://github.com/Blub/netevent"
+pkgver=2.2-beta-1
```

Comment:
opkg considers `2.2-beta-1` newer than `2.2-1`, which may be an issue if this package is updated to a non-beta version number later on. `2.2-1~beta` sorts lower than `2.2-1`. To check version ordering, you can use `opkg compare-versions`:
```console
$ opkg compare-versions 2.2-beta-1 "<<" 2.2-1 && echo "lower" || echo "not lower"
not lower
$ opkg compare-versions 2.2-1~beta "<<" 2.2-1 && echo "lower" || echo "not lower"
lower
```

```suggestion
pkgver=2.2-1~beta
```

Reply:
I spoke too early: 2.2.1 is out